main.css

```diff
@@ -1,76 +1,427 @@
     text-align: center;
-    font-size: 40px;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
   }
 
   .main-header {
     color: #172b4d;
-    font-size: 40px;
     text-align: center;
-    font-family: "Inter", sans-serif;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
   }
 
   .main-icons {
     display: flex;
     justify-content: center;
-    gap: 10px;
-    margin-top: 5%;
+    gap: 20px;
+    margin-top: 36px;
     list-style: none;
   }
 
+  .my-work-title {
+    display: flex;
+    justify-content: center;
+    margin-top: 5%;
+  }
+
+  .my-work-title img {
+    display: none;
+  }
+
+  .my-work-title h2 {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    font-style: normal;
+    color: #172b4d;
+  }
+
   .about-me {
     margin-top: 5%;
     text-align: center;
     font-size: large;
-    font-family: "Inter", sans-serif;
     margin-left: 3%;
     margin-right: 3%;
   }
 
-  .main-title {
+  .projects-container {
+    display: grid;
+    grid-template-rows: 474px 474px;
+    grid-template-columns: repeat(3, 369px);
+    row-gap: 24px;
+    column-gap: 32px;
+    margin-top: 52px;
+  }
+
+  .project-card {
+    border: 2px solid #d0d9d4;
+    background-color: #ebf0ee;
+    border-radius: 8px;
     display: flex;
     flex-direction: column;
+    justify-content: end;
+  }
+
+  .single-card {
+    background-color: white;
+    text-align: center;
+    padding: 16px;
+    height: 160px;
+    border-radius: 8px;
+  }
+
+  .single-card h3 {
+    font-family: "Inter", sans-serif;
+    font-size: 24px;
+    font-weight: 600;
+    font-style: normal;
+    color: #3a4a42;
+  }
+
+  .skills-list {
+    margin-top: 5%;
+    display: flex;
+    gap: 3%;
+    list-style: none;
     justify-content: center;
+  }
+
+  .skills-list li {
+    background-color: #ebf0ee;
+    border-radius: 4px;
+    padding: 6px 10px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+  }
+
+  .see-project-button {
+    background-color: #36b37f;
+    color: white;
+    border: none;
+    border-radius: 4px;
+    padding: 15px;
+    margin-top: 5%;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+  }
+
+  .see-project-button:hover {
+    box-shadow: 0 4px 8px 0 #36b37f69;
+  }
+
+  .see-project-button:active {
+    background-color: #008552;
+  }
+
+  .personal-works {
+    display: flex;
+    flex-direction: column;
     align-items: center;
-    padding-top: 20%;
+  }
+
+  .about-me-detail {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .about-me-head {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    background-position: center;
+    font-family: "Inter", sans-serif;
+    margin-top: 192px;
+    background-image: url("images/abouthead.png");
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
Kindly, avoid the diplication of the background-position, try to check you linters errors for that